### DISCUSSION TITLE
Vuetify webpage not working

### DISCUSSION THREAD

**Discussion by loferreiranuno** [vuetifyjs/vuetify#20495] [Q&A]
https://vuetifyjs.com

Styles are not loading.

**Comment by loferreiranuno** [comment]
![image](https://github.com/user-attachments/assets/07a0df1b-de9b-40fe-9f7e-55638be2e297)

**Comment by frederikheld** [comment]
Vuetify currently has issues with their hosting. They announced on Twitter that this is being worked on.

I suppose that monetary donations might help to fix the issue, so feel free to contribute: https://vuetifyjs.com/en/introduction/sponsors-and-backers ;-)
